Package.swift

```diff
@@ -5,14 +5,18 @@ var dependencies: [Package.Dependency] = [
     .package(url: "https://github.com/apple/swift-tools-support-core.git", .exact("0.2.3")),
     .package(url: "https://github.com/apple/swift-argument-parser", from: "1.0.1"),
 ]
+var mockoloFrameworkTargetDependencies: [Target.Dependency] = [
+    .product(name: "SwiftSyntax", package: "SwiftSyntax"),
+]
 
-let swiftSyntax: Package.Dependency
-#if swift(>=5.5)
-swiftSyntax = .package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", .exact("0.50500.0"))
+#if swift(>=5.6)
+dependencies.append(.package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", .branch("release/5.6")))
```

Comment:
SwiftSyntax has been released for Swift 5.6.
https://github.com/apple/swift-syntax/releases/tag/0.50600.0

Reply:
Thanks for suggestion.